lib/tutorials/en_US/auth.md

```diff
@@ -64,94 +165,144 @@ const start = async () => {
 start();
 ```
 
-First, we define our `users` database, which is a simple object in this example. Then we define a validation function, which is a feature specific to [hapi-auth-basic](https://github.com/hapijs/hapi-auth-basic) and allows us to verify that the user has provided valid credentials.
+First, we define our `users` database, which is a simple object in this example. Then we define a validation function, which is a feature specific to [hapi-auth-basic](https://github.com/hapijs/hapi-auth-basic) and allows us to verify that the user has provided valid credentials. For this validation function, we use `Bcrypt` to compare the user provided password with the hashed password in our database.  
 
 Next, we register the plugin, which creates a scheme with the name of `basic`. This is done within the plugin via [server.auth.scheme()](/api#serverauthschemename-scheme).
```

Comment:
The link for `server.auth.scheme` does not work. The hash should be `#-serverauthschemename-scheme` (note the dash after the hashbang)